src/GregClient/Utility/FileUtilities.cs

```diff
@@ -200,7 +186,7 @@ public static void UnZip(string zipFilePath, string unzipDirectory)
                     else
                     {
                         Directory.CreateDirectory(Path.GetDirectoryName(fullPath));
-                        System.IO.Compression.ZipFileExtensions.ExtractToFile(entry, fullPath, false);
+                        entry.ExtractToFile(fullPath, false);
```

Comment:
Everything else looks good, this `ZipArchiveEntry.ExtractToFile()` call apparently could throw exception, although rarely happen, can we wrap it with catch just in case or is it wrapped upper level already? https://docs.microsoft.com/en-us/previous-versions/hh722763(v%3Dvs.110)